polkadot/runtime/common/src/crowdloan/mod.rs

```diff
@@ -1629,9 +1634,10 @@ mod tests {
 
 			// Some funds are left over
 			assert_eq!(Balances::free_balance(&account_id), 10);
-			// They wil be left in the account at the end
+			// Remaining funds will be burned
 			assert_ok!(Crowdloan::dissolve(RuntimeOrigin::signed(1), para));
-			assert_eq!(Balances::free_balance(&account_id), 10);
+			assert_eq!(Balances::free_balance(&account_id), 0);
+			assert_eq!(Balances::total_issuance(), issuance - 10);
```

Comment:
So sending tokens to the crowdloan address basically burns them?

Reply:
Yep. It already did in the past, but the currency migration messed up the references which caused :(  
Users should never directly send funds but always use the pallet functions.